src/RecipeItem.js

```diff
@@ -0,0 +1,16 @@
+import Button from "./Style/Button"
+
+function RecipeItem({ recipe, setEditingRecipe, removeRecipe }) {
+    return (
+        <div>
+            <h3>Name: {recipe.name}</h3>
+            <h3>Description: {recipe.description}</h3>
+            <h3>Ingredients:</h3>
+            {recipe.ingredients.map(ingredient => <li>{ingredient.name}</li>)}
```

Comment:
Here you're rendering a list of components but you're not passing a key attribute. If you check the chrome dev tools you'll see a warning explaining the eproblem